src/pages/public_registration/action_accordions/UpdateTicketOwnership.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  Autocomplete,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  TextField,
+  Typography,
+} from '@mui/material';
+import { useMutation, useQueries } from '@tanstack/react-query';
+import { useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchOrganization } from '../../../api/cristinApi';
+import { updateTicket } from '../../../api/registrationApi';
+import { setNotification } from '../../../redux/notificationSlice';
+import { RootState } from '../../../redux/store';
+import { Organization } from '../../../types/organization.types';
+import { Ticket } from '../../../types/publication_types/ticket.types';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { getLanguageString } from '../../../utils/translation-helpers';
+
+interface UpdateTicketOwnershipProps {
+  ticket: Ticket;
+}
+
+export const UpdateTicketOwnership = ({ ticket }: UpdateTicketOwnershipProps) => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const customer = useSelector((state: RootState) => state.customer);
+
+  const [showUpdateOwnershipDialog, setShowUpdateOwnershipDialog] = useState(false);
+  const toggleUpdateOwnershipDialog = () => setShowUpdateOwnershipDialog(!showUpdateOwnershipDialog);
+
+  const [selectedOrganization, setSelectedOrganization] = useState<Organization | null>(null);
+
+  const allowedInstitutionIds = ticket.availableInstitutions ?? [];
+
+  const organizationQueries = useQueries({
+    queries: allowedInstitutionIds.map((orgId) => ({
+      queryKey: ['organization', orgId],
+      enabled: !!orgId,
+      queryFn: () => fetchOrganization(orgId),
+      meta: { errorMessage: t('feedback.error.get_institution') },
+      staleTime: Infinity,
+      gcTime: 1_800_000,
+    })),
+  });
+
+  const changeTicketOwnership = useMutation({
+    mutationFn: (institutionId: string) =>
+      updateTicket(ticket.id, { type: 'UpdateTicketOwnershipRequest', ownerAffiliation: institutionId }),
+    onSuccess: () =>
+      dispatch(setNotification({ message: t('feedback.success.ticket_ownership_updated'), variant: 'success' })),
+    onError: () =>
+      dispatch(setNotification({ message: t('feedback.error.ticket_ownership_update_failed'), variant: 'error' })),
+  });
+
+  const isPending = organizationQueries.some((query) => query.isPending);
+  const allowedOrganizations = isPending ? [] : organizationQueries.map((query) => query.data);
+
+  return (
+    <section>
+      <Typography fontWeight="bold">{t('registration.public_page.tasks_panel.move_task')}</Typography>
```

Comment:
Dette skulle vært en h-tag under publisering, men så lenge sidemenyen ikke har noen heading selv blir dette en litt større sak å rette opp. Viderefører derfor bare praksisen som har vært brukt så langt, så har vi egen a11y-oppgave (epic) for å rette opp sånt